addons/cargo/functions/fnc_addCargoItem.sqf

```diff
@@ -26,4 +26,4 @@ for "_i" from 1 to _amount do {
 };
 
 // Invoke listenable event
-["ace_cargoAddedByClass", [_itemClass, _vehicle, _amount]] call CBA_fnc_globalEvent;
+["ace_cargoByClassAdded", [_itemClass, _vehicle, _amount]] call CBA_fnc_globalEvent;
```

Comment:
@jonpas, as far as I understand the current code you can "add" cargo by classname, or "load" cargo by providing an object. I think the verbs are enough to differenciate both things.